QuestionAnswering/QA_ICL/backend/README.md

```diff
@@ -1,29 +1,64 @@
-# Marie Backend
+# Marie's Backend
+
+## Table of Contents
+
+- [Marie's Backend](#maries-backend)
+  - [Table of Contents](#table-of-contents)
+  - [Overview](#overview)
+  - [Architecture](#architecture)
+  - [Deployment](#deployment)
+    - [Prerequisites](#prerequisites)
+    - [Steps](#steps)
+  - [Usage](#usage)
+
+
+## Overview
+
+Marie's Backend is designed to handle natural language queries for chemistry data and provide search capabilities for chemical species and zeolites.
 
 ## Architecture
 
+The backend consists of two main components:
+
 - [`fastapi_app`](./fastapi_app/) exposes APIs for:
   - querying chemistry data via natural language
-  - searching for chemical species and zeolites based on some filtering criteria
-- [`triton_inference_server`](./triton_inference_server/) serves Sentence-BERT model.
+  - searching for chemical species and zeolites based on specific filtering criteria
+- [`triton_inference_server`](./triton_inference_server/) serves the Sentence-BERT model for generating text embeddings.
+
 
-## Deployment with Docker (recommended)
+## Deployment
```

Comment:
The deployment steps in the main README is for deploying the whole architecture, but the steps in this README is specifically for Marie's backend. This will be useful for developers who work solely on the backend and need to deploy it locally for testing. My suggestion would thus be to keep this section, but @sandradeng20 and @jlai2021 can weigh in and decide as well as you are taking over the code.

Reply:
Thank you for your detailed feedback. I've made changes related the the clarification of the backend architecture and component relationships also restructured deployment section for better clarity.